modules/apps/27-interchain-accounts/controller/keeper/keeper.go

```diff
@@ -32,17 +32,17 @@ type Keeper struct {
 
 // NewKeeper creates a new interchain accounts controller Keeper instance
 func NewKeeper(
-	cdc codec.BinaryCodec, key sdk.StoreKey,
+	cdc codec.BinaryCodec, storePrefix string, icaKeeper icakeeper.Keeper,
 	ics4Wrapper types.ICS4Wrapper, channelKeeper types.ChannelKeeper, portKeeper types.PortKeeper,
-	accountKeeper types.AccountKeeper, scopedKeeper capabilitykeeper.ScopedKeeper, msgRouter *baseapp.MsgServiceRouter,
+	scopedKeeper capabilitykeeper.ScopedKeeper, msgRouter *baseapp.MsgServiceRouter,
 ) Keeper {
 	return Keeper{
-		storeKey:      key,
 		cdc:           cdc,
+		storePrefix:   storePrefix,
```

Comment:
prefix used to create a prefixed store from the ICAKeeper

So, one store, but each ica submodule has its own namespace. This is actual quite useful. Now we can create many controller modules (currently require a new keeper for each)